tests/functional/test_users.py

```diff
@@ -0,0 +1,21 @@
+api_prefix = "/api/v1/"
+
+def test_login_page(test_client):
+    response = test_client.get(api_prefix+'/login')
+    assert response.status_code == 200
+    response = test_client.post(api_prefix+'/login')
+    assert response.status_code == 201
```

Comment:
`self.assertEqual(response.status_code, 201)` would be much better than `assert response.status_code == 201`

Reply:
In the pytest doc they state that it only uses assert but doesnt use assertEqual or the variants used in unittest

Reply:
Okay, will be on the lookout for that.